packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx

```diff
@@ -137,16 +136,14 @@ export function SquadNotificationsModal({
             Notify me about new members
           </Switch>
         )}
-        <p className="flex flex-row typo-callout text-theme-label-tertiary">
+        <p className="typo-callout text-theme-label-tertiary">
           For more options, go to your
-          <ClickableText
```

Comment:
Using this doesn't break the words no matter what CSS property I applied due to the Button's base classes.

Reply:
Yeah I also had some issues with this component, for now your solution is good.
Maybe we can add a backlog task for a new component?

Reply:
Logged! 🚢